src/components/button/Button.stories.tsx

```diff
@@ -58,7 +61,29 @@ storiesOf("Button", module)
     </div>
     )).add("Icon Buttons", () => (
         <div>
-            <Button key="link" link onClick={action("delete click")} icon={{shape: "trash"}}/>
-            <Button key="link" link onClick={action("copy click")} icon={{shape: "copy"}}/>
+            <Button key="link" onClick={action("delete click")} icon={{shape: "trash"}}/>
+            <Button key="link" onClick={action("copy click")} icon={{shape: "copy"}}/>
+            <Button key="link" link onClick={action("window-close click")} icon={{shape: "window-close"}}/>
+            <Button key="link" onClick={action("refresh click")} icon={{shape: "refresh"}}/>
+            <Button key="link" primary onClick={action("settings click")} icon={{shape: "settings"}}/>
+            <Button key="link" onClick={action("refresh click")} className="icon-button-border" icon={{shape: "refresh"}}>
+                Secondary
+            </Button>
+            <Button key="link" primary onClick={action("settings click")} className="icon-button-background" icon={{shape: "settings"}}>
+                Primary
+            </Button>
```

Comment:
we should not use CSS class like this. This CSS should get applied through Button.scss file